client/src/components/admin/AdminNews.vue

```diff
@@ -119,15 +137,26 @@ const fetchNews = async () => {
     });
   }
 };
-// Base-date-picker needs this
-const transfromToDateObjects = (news) => {
-  news.forEach((element) => {
-    element.from = adjustDateToLocal(element.from);
-    element.till = adjustDateToLocal(element.till);
-  });
+const fetchEmails = async () => {
+  try {
+    disableNewsLetter.value = true;
+    const res = await ApiService.getUserEmails(includeAllUserEmails.value);
+    userEmails.value = res.data;
+  } catch (error) {
+    console.log(error);
+    router.push({
+      name: "NetworkError",
+    });
+  } finally {
+    disableNewsLetter.value = false;
+  }
 };
 
-await fetchNews();
+await Promise.all([fetchNews(), fetchEmails()]);
+
+const onIncludeAllChanged = () => {
+  fetchEmails();
```

Comment:
The array should be emptied before the new fetch. Otherwise because there is no success indicator it would be possibly to check or uncheck and immediately clicking the send mail button while the fetch is not yet resolved and thereby sending the mails to the "wrong" recipients.

Edit: not really the case b/c of `disableNewsLetter` but an indicator would be nice anyway

Reply:
done